sdk/AWSXRayRecorder.nuspec

```diff
@@ -15,44 +15,44 @@
     <tags>AWS Amazon cloud AWSXRay XRay</tags>
     <dependencies>
       <group targetFramework=".NETFramework4.5">
-        <dependency id="AWSXRayRecorder.Core" version="2.12.0" />
-        <dependency id="AWSXRayRecorder.Handlers.AspNet" version="2.9.0" />
-        <dependency id="AWSXRayRecorder.Handlers.AwsSdk" version="2.10.0" />
-        <dependency id="AWSXRayRecorder.Handlers.EntityFramework" version="1.3.0" />
-        <dependency id="AWSXRayRecorder.Handlers.SqlServer" version="2.9.0" />
-        <dependency id="AWSXRayRecorder.Handlers.System.Net" version="2.9.0" />
+        <dependency id="AWSXRayRecorder.Core" version="2.13.0" />
+        <dependency id="AWSXRayRecorder.Handlers.AspNet" version="2.10.0" />
+        <dependency id="AWSXRayRecorder.Handlers.AwsSdk" version="2.11.0" />
+        <dependency id="AWSXRayRecorder.Handlers.EntityFramework" version="1.4.0" />
+        <dependency id="AWSXRayRecorder.Handlers.SqlServer" version="2.10.0" />
+        <dependency id="AWSXRayRecorder.Handlers.System.Net" version="2.10.0" />
       </group>
       <group targetFramework=".NETFramework4.6.2">
-        <dependency id="AWSXRayRecorder.Core" version="2.12.0" />
-        <dependency id="AWSXRayRecorder.Handlers.AspNet" version="2.9.0" />
-        <dependency id="AWSXRayRecorder.Handlers.AwsSdk" version="2.10.0" />
-        <dependency id="AWSXRayRecorder.Handlers.EntityFramework" version="1.3.0" />
-        <dependency id="AWSXRayRecorder.Handlers.SqlServer" version="2.9.0" />
-        <dependency id="AWSXRayRecorder.Handlers.System.Net" version="2.9.0" />
+        <dependency id="AWSXRayRecorder.Core" version="2.13.0" />
+        <dependency id="AWSXRayRecorder.Handlers.AspNet" version="2.10.0" />
+        <dependency id="AWSXRayRecorder.Handlers.AwsSdk" version="2.11.0" />
+        <dependency id="AWSXRayRecorder.Handlers.EntityFramework" version="1.4.0" />
+        <dependency id="AWSXRayRecorder.Handlers.SqlServer" version="2.10.0" />
+        <dependency id="AWSXRayRecorder.Handlers.System.Net" version="2.10.0" />
       </group>
       <group targetFramework=".NETStandard2.0">
-        <dependency id="AWSXRayRecorder.Core" version="2.12.0" />
-        <dependency id="AWSXRayRecorder.Handlers.AspNetCore" version="2.9.0" />
-        <dependency id="AWSXRayRecorder.Handlers.AwsSdk" version="2.10.0" />
-        <dependency id="AWSXRayRecorder.Handlers.EntityFramework" version="1.3.0" />
-        <dependency id="AWSXRayRecorder.Handlers.SqlServer" version="2.9.0" />
-        <dependency id="AWSXRayRecorder.Handlers.System.Net" version="2.9.0" />
+        <dependency id="AWSXRayRecorder.Core" version="2.13.0" />
+        <dependency id="AWSXRayRecorder.Handlers.AspNet" version="2.10.0" />
```

Comment:
Why is the dependency changed from `AspNetCore` to `AspNet`? 
We only build `AspNet` for the .Net Frameworks 4.5 and 4.6. For others, we build `AspNetCore`.